Offseason2022/src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -59,7 +59,7 @@ public Vision( )
     SmartDashboard.putNumber("VI_OVERRIDE_TY", 0.0);
     SmartDashboard.putNumber("VI_OVERRIDE_TA", 0.0);
     SmartDashboard.putNumber("VI_OVERRIDE_TS", 0.0);
-    SmartDashboard.putBoolean("VI_OVERRIDE_TV", false);
+    SmartDashboard.putNumber("VI_OVERRIDE_TV", 0.0);

```

Comment:
Can I ask what the benefit of putting a numerical value to the SmartDashboard is in this case?

Reply:
Great question. When Reeva debugged the reason the limelight driving algorithm was not working, she found that the value in network tables for "tv" (target valid) was actually a double (uses method putNumber). We had it coded to assume that value was a boolean, and that's why the code did not work. Because when someone called getBoolean to read the value, it was always false (it didn't understand the data type of "tv"). 

Soooo.... I looked at the limelight simulation code in the project, and it was also incorrectly assuming that "tv" was a boolean. I changed that for the simulation--and the override values listed above. That is what this change is doing--initializing a Smartdashboard widget named VI_OVERRIDE_TV as 0.0. 

All because the limelight guys decided to store all of the values as doubles instead of making "targetValid" be a boolean.